### PR TITLE
Il 01 list of cryptocurrencie

### PR DESCRIPTION
**Pull request by ivan-labra**

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { HeaderConteiner, TittleText, UserImage } from './styles';
+
+const Header = (): JSX.Element => (
+	<HeaderConteiner>
```
Comment (VicenBayugar) [inline src/components/Header/index.tsx]: container*

File: src/components/Header/styles.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    color: ${theme.white};
+    font-size: 24px;
+    font-weight: 700;
+    left: -35%
+`;
+
+export const UserImage = Styled.Image`
+    width: 50px;
+    height: 50px;
+    left:186%;
```
Comment (tomasdematos) [inline src/components/Header/styles.ts]: you are not using correctly the flex if you have to put this margin

Comment (tomasdematos) [reply src/components/Header/styles.ts]: try justify-content?

File: src/components/ViewCrypto/index.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Arrow_green from '../../assets/arrow_Green.png';
```
Comment (tomasdematos) [inline src/components/ViewCrypto/index.tsx]: use camelCase

File: src/components/ViewCrypto/index.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    <DataConteiner>
+      <Data>
+        <TextTitle>{item.name}</TextTitle>
+        <TextTitle>${item.price}</TextTitle>
+      </Data>
+      <Data>
+        <CodeCrypto>{item.symbol}</CodeCrypto>
+        <ChangeContainer>
+          {item.dollar_percentage_change > 0 ? (
+            <ImgArrow source={Arrow_green} />
```
Comment (tomasdematos) [inline src/components/ViewCrypto/index.tsx]: <ImgArrow source={item.dollar_percentage_change > 0 ? Arrow_green : Arrow_red} />

File: src/components/ViewCrypto/styles.ts [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    justify-content:center    
+`;
+
+export const Data = Styled.View`
+    flex-direction: row;
+    justify-content: space-between;
+`;
+
+export const TextTitle = Styled.Text`
+    color:black;
```
Comment (tomasdematos) [inline src/components/ViewCrypto/styles.ts]: use theme

File: src/screens/Home/Home.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+const Home = (): JSX.Element => {
+  const dataCryptos: Crypto[] = useAppSelector(
+    (state: RootState) => state.cryptos.cryptos,
+  );
+  return (
+    <SafeAreaView>
+      <CryptoList
+        data={dataCryptos}
+        keyExtractor={(item: Crypto) => item.id}
```
Comment (tomasdematos) [inline src/screens/Home/Home.tsx]: unnecessary type declaration, FlatList already know what type is it. and use destructuring

keyExtractor={({id}) => id}

File: src/screens/Home/Home.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+        renderItem={({ item }: { item: Crypto }) => <ViewCrypto item={item} />}
```
Comment (tomasdematos) [inline src/screens/Home/Home.tsx]: move to a render function

**Pull request merged**